softwerkskammer/lib/groups/groupMapper.js

```diff
@@ -0,0 +1,13 @@
+const conf = require('simple-configure');
```

Comment:
Bitte hier analog Nicoles Vorschlag das ganze in den Group-Konstruktor inlinen. Beispiel aus member.js fillFromUI:
    ['notifyOnWikiChanges'].forEach(property => {
      this.state[property] = !!object[property];
    });

btw.: wenn body.contactTheOrganizers nicht 'on' ist, ist es insbesondere auch nicht vorhanden!